api/open_saves.proto

```diff
@@ -430,6 +441,14 @@ message SortOrder {
   string user_property_name = 4;
 }
 
+message RecordResponse {
+  // Record fetched from Datastore.
+  Record record = 1;
+
+  // Corresponding store key of record.
+  string store_key = 2;
```

Comment:
How important is it that we return the store key here? If we don't include this, I would like the above `Record` field to be a repeated field so we can return more records at once, reducing the number of calls to `stream.Recv` / `stream.Send`.

Reply:
I needed to kep the store_key as with the record to allow for converting a protobuf from the open saves record.FromProto function.
I ended up creating an entity object to help in managing the data returned to the client.